doc/nrf/ug_thingy53.rst

```diff
@@ -0,0 +1,255 @@
+.. _ug_thingy53:
+
+Working with Thingy:53
+######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Nordic Thingy:53 is a battery-operated prototyping platform for IoT Systems.
+
+Thingy:53 integrates the following components:
+
+* nRF5340 SoC - Supporting Bluetooth® Low Energy, 802.14.5 based protocols and Near Field Communication (NFC).
+* nRF21540 RF FEM - Range extender with integrated power amplifier (PA)/low noise amplifier (LNA).
```

Comment:
low-noise amplifier
And actually I see that it is written in title case (Low-Noise Amplifier) in the Glossary, but I am not sure why..?

Reply:
I'll keep it lower case for now, and we can look into that in the improvements phase.